src/Pages/Templates/TemplatesTable/TemplatesTable.tsx

```diff
@@ -286,7 +287,11 @@ const TemplatesTable = () => {
                       <Td>{description}</Td>
                       <Td>{archesDisplay(arch)}</Td>
                       <Td>{versionDisplay(version)}</Td>
-                      <Td>{use_latest ? 'Use latest' : formatDateDDMMMYYYY(date)}</Td>
+                      <Td>
+                        <Tooltip content={formatDateUTC(date)} position='top-start' enableFlip>
```

Comment:
we may not want to show the UTC time as a tooltip if a template is set to use latest snapshots. you can use the custom ConditionalTooltip for this (like the one on line 296)

Reply:
don't know how have I missed that 🙈 fixed 👍🏼